File: api/src/main/java/com/example/api/models/Student.java
package com.example.api.models;

import jakarta.persistence.*;
import lombok.Data;
@Entity
@Table(name = "student")
@Data //Genera los getters and setters
public class Student {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @Column(name = "name", nullable = false)
    private String name;

    @Column(name = "lastname", nullable = false)
    private String lastname;

    @Column(name = "email", nullable = false)
    private String email;
}

File: client/src/app/models/Student.ts
export class Student {
    id: number;
    name: string;
    lastname: string;
    email: string;
    constructor(
        id: number = 0, 
        name: string = '', 
        latname: string = '', 
        email: string = '', 
    ) {
        this.id = id;
        this.name = name;
        this.lastname = latname;
        this.email =email;
    }
  }
  